server/connectors/slack_connector/client.py

```
@@ -48,84 +82,145 @@ def _validate_response(self, result: Dict[str, Any], endpoint: str) -> Dict[str,
                 logger.exception("Request to Slack API failed on %s", endpoint)
                 raise ValueError(f"Failed to communicate with Slack: {e}") from e
-        
-        raise ValueError(f"Failed to communicate with Slack after {max_retries} retries: rate limited on {endpoint}")
-    
-    def send_message(self, channel: str, text: str, thread_ts: Optional[str] = None, 
-                     blocks: Optional[List[Dict]] = None) -> Dict[str, Any]:
+
+        raise ValueError(
+            f"Failed to communicate with Slack after {max_retries} retries: "
+            f"rate limited on {endpoint}"
+        )
```

Ruff (0.15.15)

[warning] Avoid specifying long messages outside the exception class

(TRY003)